graph/docu/docu_04graph_02plotutils.cc

```diff
@@ -51,9 +56,15 @@ TEST(Graph, PlotutilsLove) {
     .X("... living life in peace")
     .Y("John Lennon, \"Imagine\"")
     .LineWidth(0.015)
-    .OutputFormat("png");
-
-ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString("golden/love.png"));
+    .BitmapSize(800, 800)
+#if 1      
```

Comment:
Yes for both the test to pass, for the code to generate a "png", and the documentation to look clean and complete.